src/styles/block/calendar.scss

```diff
@@ -0,0 +1,49 @@
+.calendar {
+  display: flex;
+  justify-content: center;
+  align-self: center;
+
+  &__day {
+    text-align: center;
+    line-height: $text-line-height;
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 31px;
+    background-color: grey;
+    transition: background-color 0.5s, transform 0.5s;
```

Comment:
move transition time to variable and use here

Reply:
done